codility_training/lessons.lesson10.PrimeAndCompositeNumbers.CountFactors/NowonLee.js

```diff
@@ -0,0 +1,16 @@
+function solution(N) {
+    if(N===1) return 1
+    var result = 0;
```

Comment:
```suggestion
    let result = 0;
```
변수는 `let` 상수는 `const`